app/pages/DomainManager/index.js

```diff
@@ -22,9 +25,32 @@ class DomainManager extends Component {
     this.props.getMyNames();
   }
 
+  handleExportClick() {
+    let names = this.props.myDomains.map(({name}) => name);
+    let data = names.join("\n");
+
+    let savePath = dialog.showSaveDialogSync({
+      filters: [{ name: 'spreadsheet', extensions: ['csv'] }]
+    });
+
+    if(savePath) {
+      fs.writeFile(savePath, data, (err) => {
+        if (err) {
+          throw err;
```

Comment:
can you add error handling here? you can refer to the example here: https://github.com/kyokan/bob-wallet/blob/master/app/pages/Auction/BidActionPanel/Reveal.js#L61-L63

you will also have to add a new md2p: https://github.com/kyokan/bob-wallet/blob/master/app/pages/Auction/BidActionPanel/Reveal.js#L193

Reply:
@chikeichan - yep the sendReveal example makes sense

what is an md2p? are we making schedule I substances?